app/src/main/kotlin/com/intive/picover/parties/viewmodel/PartyDetailsViewModel.kt

```diff
@@ -0,0 +1,29 @@
+package com.intive.picover.parties.viewmodel
+
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.lifecycle.SavedStateHandle
+import com.intive.picover.common.viewmodel.LifecycleViewModel
+import com.intive.picover.parties.repository.MockedPartiesRepository
+import com.intive.picover.parties.state.PartyDetailsState
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class PartyDetailsViewModel @Inject constructor(
+	private val partiesRepository: MockedPartiesRepository,
```

Comment:
should be probably re-ordered to:
...
savedStateHandle: SavedStateHandle,
private val partiesRepository: MockedPartiesRepository,
...
 
as we want to stick to ordering we already discussed

Reply:
Done